src/types/type-helpers.ts

```diff
@@ -29,3 +34,10 @@ export function upgradeCurrencyCode(opts: {
 
   return { currencyCode, tokenId: tokenId ?? null }
 }
+
+export function upgradeSwapQuote(quote: EdgeSwapQuote): EdgeSwapQuote {
+  if (quote.networkFee != null && quote.networkFee.tokenId == null) {
+    quote.networkFee.tokenId = quote.request.fromTokenId
+  }
```

Comment:
I would have put this code directly into `fetchSwapQuotes`, rather than creating a separate method, since that's less cognitive load. Optional (but recommended) change, since this way works too.